src/rules/russian.toml

```diff
@@ -0,0 +1,78 @@
+min_trimmed_length = 3
+
+# For count = 2 we have sentences like: "Это он", "Будет беда"
+# and a lot of trash abbreviations like "до н.", "и доп."
+min_word_count = 3
+
+max_word_count = 14
+
+# In Russian, words can consist of one letter, so no restrictions
+min_characters = 0
+
+may_end_with_colon = false
+quote_start_with_letter = true
+needs_punctuation_end = false
+needs_letter_start = true
+
+# Apparently, in some places the sentences are cut incorrectly,
+# which is why we get some part of the sentence, and not its entirety.
+# This is required to fix sentences like:
+# с Иваном Галамяном.
+needs_uppercase_start = true
+
+allowed_symbols_regex = "[А-Яа-яёЁ\\s:,.\\-‑?;!—­‐–'·=’―−”‘]"
+
+disallowed_symbols = [
```

Comment:
Note that this won't be used as `allowed_symbols_regex` is set. However, given that those invisible chars should not be allowed by the above regex, I guess that's fine and it can be completely removed?

Reply:
No. As I found out invisible chars are not detected by regex for some reason. Perhaps because they are part of `\s`. Therefore, they had to be left.

Reply:
yeah, that's probably what is happening here. Could you just use the specific character you want for whitespace instead of `\s` or is my Regex knowledge completely failing me here?